src/views/PaymentNetworkFees.vue

```diff
@@ -0,0 +1,202 @@
+
+  if (discount > 0) {
+    price = price - (price / 100) * discount;
+  }
+
+  return price;
+}
+
+function computePlanFiatPrice(plan, unit?: number): number {
+  return computePrice(null, unit || plan.unit, plan.discount);
+}
+
+function formatFiatCurrency(amount: number): string {
+  return `${BASE_CURRENCY.name} ${
+    BASE_CURRENCY.symbol
+  }${amount.toLocaleString()}`;
+}
+
+function formatCryptoCurrency(amount: number, currency): string {
+  return `~${currency.code} ${amount.toLocaleString()}`;
+}
+
+watch(paymentTx, () => {
+  modalPostPaymentOpen.value = !!paymentTx.value;
+});
+</script>
+
+<template>
+  <TheLayout class="max-w-lg px-4 md:px-0">
+    <BaseMessageBlock
+      v-if="!web3Account"
+      level="warning"
+      class="mb-3"
+      is-responsive
+    >
+      Connect your wallet first to proceed
+    </BaseMessageBlock>
+
+    <h1>Network fees</h1>
+    <div class="text-lg mb-4">Pay to support your network on Snapshot</div>
+
+    <form v-if="fxLoadStatus === 1" class="flex flex-col gap-4" @submit="pay">
+      <fieldset>
+        <legend>
+          <h2 class="text-lg mb-2">Select your package</h2>
+        </legend>
+        <ol class="flex flex-col gap-2">
+          <li v-for="(plan, planId) in PLANS" :key="planId">
+            <label
+              class="flex gap-3 border border-skin-border rounded-xl p-3 cursor-pointer"
+              :class="{
+                '!border-skin-primary': planId === data.plan
+              }"
+              @click="setData('plan', planId)"
+            >
+              <TuneRadio :value="planId" :model-value="data.plan" />
+              <div class="flex-grow">
+                <b>{{ plan.label }}</b>
+                <BasePill
+                  v-if="plan.discount"
+                  class="ml-2 py-1 !bg-skin-primary"
+                >
+                  Save {{ plan.discount }}%
+                </BasePill>
+                <small class="block text-skin-text">
+                  {{
+                    formatFiatCurrency(computePlanFiatPrice(plan, BASE_UNIT))
+                  }}/month
+                </small>
+              </div>
+              <span>
+                {{ formatFiatCurrency(computePlanFiatPrice(plan)) }}
+              </span>
+            </label>
+          </li>
+        </ol>
+      </fieldset>
+
+      <fieldset>
+        <legend>
+          <h2 class="text-lg mb-2">Select your currency</h2>
+        </legend>
+
+        <ol class="flex flex-col gap-2">
+          <li v-for="(currency, currencyId) in CURRENCIES" :key="currencyId">
+            <label
+              class="flex border border-skin-border gap-3 rounded-xl p-3 cursor-pointer"
+              :class="{
+                '!border-skin-primary': currencyId === data.currency
+              }"
+              @click="setData('currency', currencyId)"
+            >
+              <TuneRadio :value="currencyId" :model-value="data.currency" />
+              <BaseAvatar
```

Comment:
You can remove the logo url from the token object and simply use the `AvatarToken` with the contract address